src/components/camera.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { RNCamera } from 'react-native-camera';
+import colors from '../assets/colors.js'
+
+class Camera extends React.Component {
+  render() {
+    return (
+      <View style={{flex: this.props.flex, flexDirection: 'row'}}>
+        <View style={styles.rowContainer}>
+          <RNCamera
+            ref={ref => {
+              this.camera = ref;
+            }}
+            style = {styles.preview}
+            type={RNCamera.Constants.Type.back}
+            flashMode={RNCamera.Constants.FlashMode.on}
```

Comment:
is this flash as in camera flash? I think there's a good chance that flash will make the picture harder for ML to recognize. Ask the ML team